PairedEndSingleSampleWf-fc-b37.wdl

```diff
@@ -0,0 +1,1753 @@
+## Copyright Broad Institute, 2017
+##
+## This WDL pipeline implements data pre-processing and initial variant calling (GVCF
+## generation) according to the GATK Best Practices (June 2016) for germline SNP and
+## Indel discovery in human whole-genome sequencing (WGS) data.
+##
+## Requirements/expectations :
+## - Human whole-genome pair-end sequencing data in unmapped BAM (uBAM) format
+## - One or more read groups, one per uBAM file, all belonging to a single sample (SM)
+## - Input uBAM files must additionally comply with the following requirements:
+## - - filenames all have the same suffix (we use ".unmapped.bam")
+## - - files must pass validation by ValidateSamFile
+## - - reads are provided in query-sorted order
+## - - all reads must have an RG tag
+## - GVCF output names must end in ".g.vcf.gz"
+## - Reference genome must be Hg38 with ALT contigs
+##
+## Runtime parameters are optimized for Broad's Google Cloud Platform implementation.
+## For program versions, see docker containers.
+##
+## LICENSING :
+## This script is released under the WDL source code license (BSD-3) (see LICENSE in
+## https://github.com/broadinstitute/wdl). Note however that the programs it calls may
+## be subject to different licenses. Users are responsible for checking that they are
+## authorized to run all programs before running this script. Please see the docker
+## page at https://hub.docker.com/r/broadinstitute/genomes-in-the-cloud/ for detailed
+## licensing information pertaining to the included programs.
+
+# WORKFLOW DEFINITION
+workflow PairedEndSingleSampleWorkflow {
+
+  File contamination_sites_ud
+  File contamination_sites_bed
+  File contamination_sites_mu
+  File? fingerprint_genotypes_file
+  File? fingerprint_genotypes_index
+  File? haplotype_database_file
+  File wgs_evaluation_interval_list
+  File wgs_coverage_interval_list
+
+  String sample_name
+  String base_file_name
+  String final_gvcf_base_name
+  File flowcell_unmapped_bams_list
+  Array[File] flowcell_unmapped_bams = read_lines(flowcell_unmapped_bams_list)
+  String unmapped_bam_suffix
+
+  File wgs_calling_interval_list
+  Int haplotype_scatter_count
+  Int break_bands_at_multiples_of
+  Int? read_length
+
+  File ref_fasta
+  File ref_fasta_index
+  File ref_dict
+  File? ref_alt
+  File ref_bwt
+  File ref_sa
+  File ref_amb
+  File ref_ann
+  File ref_pac
+
+  File dbSNP_vcf
+  File dbSNP_vcf_index
+  Array[File] known_indels_sites_VCFs
+  Array[File] known_indels_sites_indices
+
+  Int preemptible_tries
+  Int agg_preemptible_tries
+
+  String? gotc_docker_override
```

Comment:
What does gotc stand for? Should these names be gotc -> gitc?

Reply:
"Genome On The Cloud" 
still used here for gcr version 
https://console.cloud.google.com/gcr/images/broad-gotc-prod/US/genomes-in-the-cloud?gcrImageListsize=30

I think i can change if you suggest it

Reply:
I thought it's called Genomes IN the cloud in other workflows, so I would change it